test/k4FWCoreTest/src/components/ExampleEventHeaderCheck.cpp

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
It's a bit overkill to create another algorithm just to check that the event numbers are a permutation of some numbers, no? We could test this with the python script to test outputs.

Reply:
Yeah I went overboard with this. Moved testing to `checkOutputFiles.py`